### PR TITLE
Add more new line chars to N to filter out

### PR DESCRIPTION
**Pull request by raronson**

### CODE REVIEW

File: src/main/scala/com/ambiata/disorder/N.scala [hunk 1]
```diff
@@ -11,9 +11,12 @@ import GenPlus._
 case class N(value: String)
 
 object N {
+
+  val newlines: List[Char] = List('\n', '\r', 0x000B, 0x000C, 0x0085, 0x2028, 0x2029)
```
Comment (raronson) [inline src/main/scala/com/ambiata/disorder/N.scala]: I got this list from http://en.wikipedia.org/wiki/Newline#Unicode

File: src/main/scala/com/ambiata/disorder/N.scala [hunk 2]
```diff
@@ -11,9 +11,12 @@ import GenPlus._
...
+
   implicit def NArbitrary: Arbitrary[N] =
     Arbitrary(for {
       a <- arbitrary[S]
-      c <- checkGen[Char](arbitrary[Char], "Char", x => x != '\n' && x != '\r')
-    } yield N(a.value.replace('\n', c).replace('\r', c)))
+      c <- checkGen[Char](arbitrary[Char], "Char", x => !newlines.contains(x))
+    } yield N(newlines.foldLeft(a.value)({ case (b, a) => b.replace(a, c) })))
```
Comment (charleso) [inline src/main/scala/com/ambiata/disorder/N.scala]: There isn't a way of replacing them all in one pass is there by any happy chance? /nit

**Comment by charleso** [comment]
:+1:

File: src/main/scala/com/ambiata/disorder/N.scala (hunk 2 above)
Comment (raronson) [inline src/main/scala/com/ambiata/disorder/N.scala]: Im not sure how, suggestions?

Comment (charleso) [inline src/main/scala/com/ambiata/disorder/N.scala]: > Im not sure how, suggestions?

Not without using a regex. :(

**Pull request merged**